rules/no-truthy-falsy.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const {
+  getDocsUrl,
+  expectCase,
+  expectNotCase,
+  expectResolveCase,
+  expectRejectCase,
+  method,
+} = require('./util');
+
+module.exports = {
+  meta: {
+    docs: {
+      url: getDocsUrl(__filename),
+    },
+    fixable: 'code',
+  },
+  create(context) {
+    return {
+      CallExpression(node) {
+        if (
+          !(
+            expectNotCase(node) ||
+            expectResolveCase(node) ||
+            expectRejectCase(node)
+          ) &&
+          expectCase(node)
+        ) {
+          if (method(node).name === 'toBeTruthy') {
+            context.report({
+              message: 'Use toBe(true) instead',
+              node: method(node),
+            });
+          } else if (method(node).name === 'toBeFalsy') {
```

Comment:
can you extract `method(node)` into a variable?

Reply:
Unsure if you meant something in utils or here locally. Happy to adjust.

Reply:
Locally 😀 
38f3ec57dde973e4c3f617174438d6e1d1822c4d